packages/outline-view/src/browser/outline-view-widget.tsx

```diff
@@ -62,15 +63,16 @@ export class OutlineViewWidget extends TreeWidget {
         this.addClass('theia-outline-view');
     }
 
-    public setOutlineTree(roots: OutlineSymbolInformationNode[]) {
+    public setOutlineTree(roots: OutlineSymbolInformationNode[]): void {
         const nodes = this.reconcileTreeState(roots);
-        this.model.root = {
+        this.root = {
```

Comment:
I was trying to find a way to get the `children` property since it exists in the `CompositeTreeNode` and not `TreeNode`. I'll have to look more closely on how to do so.

Reply:
thanks, I'll make the necessary changes :)